pkg/kubectl/describe.go

```diff
@@ -1395,6 +1396,32 @@ func DescribeEvents(el *api.EventList, w io.Writer) {
 	}
 }
 
+// DeploymentDescriber generates information about a deployment.
+type DeploymentDescriber struct {
+	client.Interface
+}
+
+func (dd *DeploymentDescriber) Describe(namespace, name string) (string, error) {
+	d, err := dd.Experimental().Deployments(namespace).Get(name)
+	if err != nil {
+		return "", err
+	}
+	return tabbedString(func(out io.Writer) error {
+		fmt.Fprintf(out, "Name:\t%s\n", d.ObjectMeta.Name)
+		fmt.Fprintf(out, "Namespace:\t%s\n", d.ObjectMeta.Namespace)
+		fmt.Fprintf(out, "CreationTimestamp:\t%s\n", d.CreationTimestamp.Time.Format(time.RFC1123Z))
+		fmt.Fprintf(out, "Labels:\t%s\n", labels.FormatLabels(d.Labels))
+		fmt.Fprintf(out, "Selector:\t%s\n", labels.FormatLabels(d.Spec.Selector))
+		fmt.Fprintf(out, "Replicas:\t%d updated / %d total\n", d.Status.UpdatedReplicas, d.Spec.Replicas)
+		fmt.Fprintf(out, "StrategyType:\t%s\n", d.Spec.Strategy.Type)
+		if d.Spec.Strategy.RollingUpdate != nil {
+			ru := d.Spec.Strategy.RollingUpdate
+			fmt.Fprintf(out, "RollingUpdateStrategy:\t%s maxUnavailable, %s maxSurge, %d minReadySeconds", ru.MaxUnavailable.String(), ru.MaxSurge.String(), ru.MinReadySeconds)
```

Comment:
I like describe to a bit more human-friendly, so I would suggest "max unavailable", "max surge", etc.